sstables_loader.cc

```diff
@@ -620,6 +620,12 @@ class sstables_loader::download_task_impl : public tasks::task_manager::task::im
         return tasks::is_abortable::yes;
     }
 
+    virtual future<> release_resources() noexcept override {
```

Comment:
We call `release_resources` when the task is complete, but it may still be kept in task manager. If users check the status of the finished task they will see 0/0 progress. 

Please save the final progress here and adjust `get_progress()` accordingly.